File: packages/rust/tests/fixtures/unit_mutation/python/killed/tests/integration/tiers_test.py
import pytest


# The package-level integration tier. The unit mutation gate judges mutants by the scan
# path's colocated suite alone; if it ever runs this tier, the clean (baseline) run fails loudly.
def test_is_never_run_by_the_unit_mutation_gate():
    pytest.fail("the unit mutation gate ran a package-level suite tier")

File: packages/rust/tests/fixtures/unit_mutation/typescript/declaration_only/tests/integration/tiers.test.ts
import { it, expect } from 'vitest';

// The package-level integration tier. The unit mutation gate judges mutants by the scan
// path's colocated suite alone; if it ever runs this tier, the initial (dry) run fails loudly.
it('is never run by the unit mutation gate', () => {
  expect.unreachable('the unit mutation gate ran a package-level suite tier');
});
